ADB_world/def_flight_connection.py

```diff
@@ -0,0 +1,170 @@
+import json
+import pandas as pd
+import geopandas as gpd
+import os
+from shapely.geometry import LineString
+
+
+def generate_flight_connections_json(month):
```

Comment:
Here I will need a descriptive docstring. I don't want step through the entire code to figure out what the output looks like. You can take my functions here for inspiration: https://github.com/sustainableaviation/EcoPyLot/blob/8a6b7b5dff270293a38754459cb5f2812471245e/ecopylot/inout.py#L57